frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -68,11 +69,13 @@ local symbol_prefix = {
         -- @translators This is the footer letter prefix for frontlight level.
         frontlight = C_("FooterLetterPrefix", "L:"),
         -- @translators This is the footer letter prefix for frontlight warmth (redshift).
```

Comment:
Or CT for color temperature? LW seems very unintuitive to me compared to redshift or color temp. In any case the comment should clearly explain what's going on.
```suggestion
        -- @translators This is the footer letter prefix for light warmth of the frontlight (redshift).
```